man/crypttab.xml

```diff
@@ -327,6 +327,14 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>tcrypt-veracrypt</option></term>
+
+        <listitem><para>Check for a VeraCrypt volume.  Enabling this
```

Comment:
Hm, could you expand this a bit, to describe what the consequence are. (Why is it slow?, What does it do?)